src/vorta/utils.py

```diff
@@ -179,9 +175,17 @@ def choose_file_dialog(parent, title, want_folder=True):
     return dialog
 
 
-def get_private_keys():
+def is_ssh_file(filepath: str) -> bool:
+    """Check if the file is a SSH key."""
+    with open(filepath, 'rb') as f:
+        first_line = f.readline()
+    if not first_line.startswith(b'-----BEGIN OPENSSH PRIVATE KEY-----'):
```

Comment:
I just wondered whether we should only support openssh key files instead of all files starting with `-----BEGIN( \w+)? PRIVATE KEY-----`. But I guess if anyone needs that, we will see an FR opened.

Reply:
Yes, would be better. I have one SSH key here that starts with `-----BEGIN RSA PRIVATE KEY-----`. It's an older key I made for AWS EC2.

So there could be 3 different headers: https://superuser.com/a/1444343/890011

Reply:
Okay, I'll use the regex @real-yfprojects mentioned